documentation/modules/rn-2.6.adoc

```diff
@@ -163,14 +163,55 @@ When vSphere does not receive updates about the guest operating system from the
 
 The migration process fails when migrating an image-based VM from {osp} to the `default` project. link:https://issues.redhat.com/browse/MTV-964[(MTV-964)]
 
-For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12435571[Known Issues] in Jira.
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12436066[Known Issues] in Jira.
+
+// Andy, please insert the correct filter.
```

Comment:
@anarnold97 how about the following so that we won't need to update it per each z-version of 2.6?
```
project = MTV AND issuetype = Bug AND status in (New, ASSIGNED, POST, MODIFIED, ON_QA) AND component not in (Documentation, QE-Task) AND affectedVersion >= 2.0.0 AND affectedVersion < 2.7.0 ORDER BY priority DESC
```